### PR TITLE
Letter counter

### PR DESCRIPTION
**Pull request by MartaSzymula**
This function counts letters in a given text. count1 function is almot 5 times faster than count2.

### CODE REVIEW

File: letter_counter.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import timeit
+def counter1(text):
+    # """ """
+    count1 = {}
+    letters = list(string.ascii_letters)
+    sum_of_letters = 0
+    for i in letters:
+        sum_of_letters = text.count(i, 0, -1)
+        count1[i] = sum_of_letters
+    # print(count1)
```
Comment (WojciechKusa) [inline letter_counter.py]: you shouldn't add commented code to the master

File: letter_counter.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import string
+import timeit
+def counter1(text):
```
Comment (WojciechKusa) [inline letter_counter.py]: add newline between imports and function definition

File: letter_counter.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    # """ """
```
Comment (WojciechKusa) [inline letter_counter.py]: add documentation

**Review by WojciechKusa — COMMENTED** [review]
for both functions add return statements with counter

File: letter_counter.py (hunk 3 above)
Comment (MartaSzymula) [reply letter_counter.py]: done

File: letter_counter.py (hunk 2 above)
Comment (MartaSzymula) [reply letter_counter.py]: fixed

File: letter_counter.py (hunk 1 above)
Comment (MartaSzymula) [reply letter_counter.py]: fixed

**Pull request merged**